README.md

```diff
@@ -1,43 +1,48 @@
-![Flutter CI](https://github.com/canonical/ubuntu-desktop-installer/workflows/Flutter%20CI/badge.svg)
+# Ubuntu Desktop Installer
 
-# ubuntu-desktop-installer
+[![Flutter CI](https://github.com/canonical/ubuntu-desktop-installer/workflows/Flutter%20CI/badge.svg)](https://github.com/canonical/ubuntu-desktop-installer/actions)
+
+This project is a modern implementation of the Ubuntu Desktop installer, using [subiquity](https://github.com/canonical/subiquity) as a backend and [Flutter](https://flutter.dev/) for the UI.
 
 ## Checkout & Run
 
 Grab the source:
-```
+```sh
 git clone https://github.com/canonical/ubuntu-desktop-installer.git
 cd ubuntu-desktop-installer
 git submodule update --init --recursive
+cd -
```

Comment:
I wouldn't add this personally. All of the 'Checkout & Run' instructions are written such that you can run them from top to bottom. If you `cd -` it'll put you in the wrong directory for the next set of commands.

Reply:
Removed.